plugins/device/deviceutils/store-utils.go

```diff
@@ -51,14 +52,52 @@ func GetDevicesFromPeer(peerInfo *peer.Peer) ([]deviceapi.Info, error) {
 	return deviceInfo, nil
 }
 
+// SetDeviceState sets device state and updates device state in etcd
+func SetDeviceState(peerID string, deviceName string, deviceState string) error {
```

Comment:
suggestion:

if an input is of the same type, this can be simplified as below for better readability

```
func SetDeviceState(peerID, deviceName, deviceState string) error {
```